lua/schat/cl_interface.lua

```diff
@@ -1,3 +1,5 @@
+CreateClientConVar("disable_betterchat", "0", true, false)
```

Comment:
Please rename to `customchat` to avoid confusion (`betterchat` is the name of another chat addon)

Reply:
Sure I can do that for ya!